lib/install/validate/network.go

```diff
@@ -318,13 +318,8 @@ func (v *Validator) network(op trace.Operation, input *data.Data, conf *config.V
 	// we also need to have the appliance attached to the bridge network to allow
 	// port forwarding
 	conf.AddNetwork(bridgeNet)
-
-	// make sure that the bridge IP pool is large enough for bridge networks
-	err = v.checkBridgeIPRange(input.BridgeIPRange)
```

Comment:
Any reason to remove this validation? It is used by both create and configure commands. The current code structure put all validations under validate package.

Reply:
These codes only serve for create command, we do not support configure for bridge-network-range and bridge-network-width.

Reply:
Then checkBridgeIPRange is not used anymore. If so, please remove this api.